notesapi/v1/tests/test_views.py

```diff
@@ -101,12 +113,10 @@ class AnnotationViewTests(BaseAnnotationViewTests):
     Test annotation views, checking permissions
```

Comment:
I recognize that it's not new in this PR, and it's not a top priority, but I suggest it would be better to group the tests into three separate test case classes, each corresponding to the view class being tested.